e2echart/e2e-chart-template.html

```diff
@@ -400,7 +411,7 @@ <h5 class="modal-title">Resource</h5>
         })
 
         timelineGroups.push({group: "endpoint-availability", data: []})
-        createTimelineData("Failed", timelineGroups[timelineGroups.length - 1].data, eventIntervals, isEndpointConnectivity, regex)
+        createTimelineData(disruptionValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isEndpointConnectivity, regex)
 
```

Comment:
That's tricky right there passing in either a string or a function...  No issues, just made me go look.

Reply:
Yup made me look as well, turns out the function supports both. Good ol Javascript.